students/uss_tomasz/lesson_07_manipulating_strings_and_datetime_formatting/7_1/bullet_point_adder.py

```diff
@@ -0,0 +1,13 @@
+#! python3
```

Comment:
Does this shebang really work? I see 2 potential problems here:
- space between `#!` and the path
- no full path
I'm used to following form of shebang working on unix-like systems `#!/usr/bin/python3`. On windows hosts shebang doesn't work anyway.